src/diem/testing/cli/click.py

```diff
@@ -41,6 +41,8 @@ def start_server(
 
 @click.command()
 @click.option("--target", "-t", default="http://localhost:8888", help="Target mini-wallet application URL.")
+@click.option("--stub-host", "-h", default="localhost", help="Miniwallet offchain stub server host.")
```

Comment:
I think using “offchain stub server host” is not really it is, because this is just happened to be the way we construct the default vasp base url for offchain.

Thus back to my previous question, are we configuring vasp base url, or the wallet stub server host?

For example, if we can access the wallet stub server by url:  "http://mini-wallet-stub.com:port"
Then the vasp base url is: http://mini-wallet-stub.com:port
but the server start host maybe is: `0.0.0.0:port`, or `ip-address:port`

It looks like you are assuming we configure the vasp base url and server host by ip address, then we can still create a vasp-base-url from the host and port, which is fine, but I think we need to be clear it is stub server host ip address, instead of offchain server host / domain.

Reply:
given the default bind port is random, I can't pass in a url for the vasp, because I need to ensure they match. The problem here is bind host/port vs on-chain host/port. Do you have any objections to changing default bind to 0.0.0.0? then the host argument can refer specifically only to `AppConfig.client`. Does that make sense?